index.html

```diff
@@ -12,6 +12,122 @@
       <a class="menu"> Develover Tools </a>
     </div>
     <div class="page-content">
+      <div class="dev">
+        <div class="dev-header">
+          <img class="dev-header-triangles" src="images/triangle.svg">
```

Comment:
Since this is just for styling purposes, use [pseudo elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements)